### PR TITLE
[BH-1898] Fix of GUI freeze after intensive volume change

### PR DESCRIPTION
**Pull request by Lefucjusz**
Fix of the issue that GUI would freeze for
several seconds if volume level was being
changed intensively for a while.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [x] Has changelog entry added

### CODE REVIEW

File: products/BellHybrid/apps/common/include/common/models/AbstractAudioModel.hpp [hunk 1]
```diff
@@ -39,7 +39,10 @@ namespace app
         using OnPlaybackFinishedCallback = std::function<void(PlaybackFinishStatus)>;
 
         virtual ~AbstractAudioModel() noexcept                                                              = default;
-        virtual void setVolume(Volume volume, PlaybackType playbackType, OnStateChangeCallback &&callback)  = 0;
+        virtual void setVolume(Volume volume,
+                               PlaybackType playbackType,
+                               audio::VolumeUpdateType updateType = audio::VolumeUpdateType::UpdateDB,
+                               OnStateChangeCallback &&callback   = {})                                       = 0;
```
Comment (MateuszSz-mudita) [inline products/BellHybrid/apps/common/include/common/models/AbstractAudioModel.hpp:45]: Also alignment here

**Review by MateuszSz-mudita — APPROVED** [review]

File: products/BellHybrid/apps/application-bell-relaxation/presenter/RelaxationVolumePresenter.hpp [hunk 2]
```diff
@@ -24,12 +25,13 @@ namespace app::relaxation
         virtual ~AbstractRelaxationVolumePresenter()              = default;
         virtual VolumeData getVolumeData()                        = 0;
         virtual void setVolume(AbstractAudioModel::Volume volume) = 0;
+        virtual void saveVolume(AbstractAudioModel::Volume volume) = 0;
```
[the remark this answers is not in the document]
Comment (Lefucjusz) [reply products/BellHybrid/apps/application-bell-relaxation/presenter/RelaxationVolumePresenter.hpp:28]: This alignment is done automatically by `clang-format` that we've got configured for style checking - if my PR passed through all the pipeline checks then the formatting is right. I won't be able to apply the changes you requested, as they will either be automatically overwritten by the commit hook triggering `clang-format` or (if I disable the check locally) will cause my PR to fail style checking.

File: module-audio/Audio/AudioCommon.hpp [hunk 3]
```diff
@@ -82,6 +82,12 @@ namespace audio
         Enable
     };
 
+    enum class VolumeUpdateType
+    {
+        UpdateDB,
+        SkipDBUpdate
+    };
```
Comment (dawidwojtas-mudita) [inline module-audio/Audio/AudioCommon.hpp:89]: What about `SkipUpdateDB`. It looks similar to `UpdateDB`

**Review by dawidwojtas-mudita — APPROVED** [review]

File: module-audio/Audio/AudioCommon.hpp (hunk 3 above)
Comment (Lefucjusz) [reply module-audio/Audio/AudioCommon.hpp:89]: You're right, I'll change that :wink:

Comment (Lefucjusz) [reply module-audio/Audio/AudioCommon.hpp]: Done!

**Review by Maciej-Mudita — APPROVED** [review]

**Pull request merged**